pkg/services/network/nsxt_vpc_provider.go

```diff
@@ -168,7 +173,7 @@ func (vp *nsxtVPCNetworkProvider) GetClusterNetworkName(ctx context.Context, clu
 
 // The GetVMServiceAnnotations method always returns an empty map representing annotations.
 func (vp *nsxtVPCNetworkProvider) GetVMServiceAnnotations(_ context.Context, _ *vmware.ClusterContext) (map[string]string, error) {
-	return map[string]string{}, nil
+	return map[string]string{AnnotationEnableEndpointHealthCheckKey: ""}, nil
```

Comment:
Should we have a value here or is it defined that it must be empty?

```suggestion
	return map[string]string{AnnotationEnableEndpointHealthCheckKey: "true"}, nil
```

Reply:
This is expected to be empty

Reply:
Let's add a comment please that it needs to be empty :-)